packages/flutter_tools/lib/src/reporting/unified_analytics.dart

```diff
@@ -52,9 +55,25 @@ Analytics getAnalytics({
     dartVersion: flutterVersion.dartSdkVersion,
     enableAsserts: enableAsserts,
     clientIde: clientIde,
+    enabledFeatures: getEnabledFeatures(config),
   );
 }
 
+/// Uses the [Config] object to get enabled features.
+String? getEnabledFeatures(Config config) {
+  // Create string with all enabled features to send as user property
+  final Iterable<Feature> enabledFeatures =
+      allFeatures.where((Feature feature) {
+    final String? configSetting = feature.configSetting;
+    return configSetting != null && config.getValue(configSetting) == true;
```

Comment:
is this the equivalent logic to what we do with the legacy analytics?

Reply:
Yep, i borrowed the logic from [here](https://github.com/eliasyishak/flutter/blob/send-enabled-features/packages/flutter_tools/lib/src/reporting/usage.dart#L203-L215)